dev/ci/scripts/utils/ci_utils.sh

```diff
@@ -122,10 +122,19 @@ function cancel_all_batch_jobs () {
 function create_experiment () {
 
   local yaml_config="${1}"
+  local TAG="${2:-}"
```

Comment:
```suggestion
  pr_sha=$(git rev-parse --short HEAD)
  local TAG="${2:-${pr_sha}}"
```
and then delete the line for `pr_sha=` on 127